Fixed issue with translation of product options

MarekB01 opened the pull request:
https://github.com/BroadleafCommerce/QA/issues/4771

Added some logic which compares option value to key and translates product

MarekB01 committed:
BroadleafCommerce/QA-4771_locale_translation_issue
Added some logic which compares option value to key and translates product

danielcolgrove commented on core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/service/OrderItemServiceImpl.java:
```diff
@@ -20,12 +20,7 @@
-import org.broadleafcommerce.core.catalog.domain.ProductOption;
-import org.broadleafcommerce.core.catalog.domain.Sku;
-import org.broadleafcommerce.core.catalog.domain.SkuBundleItem;
+import org.broadleafcommerce.core.catalog.domain.*;
```
We don't want to use wildcard imports.

oleksiimiroshnyk commented on core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/order/service/OrderItemServiceImpl.java:
```diff
@@ -125,6 +120,13 @@ protected void populateProductOptionAttributes(OrderItem item, Map<String, Strin
                 if(value == null){
                     continue;
                 }
+                List<ProductOptionValue> optionValues = ((DiscreteOrderItem) item).getSku().getProductOptionValues();
```
I think we can extract this to some method, like String getTranslatedValue(String key) ?

Also getProductOptionValues looks like deprecated let's don't use deprecated in new code ?

MarekB01 committed:
BroadleafCommerce/QA-4771_locale_translation_issue
Added some logic which compares option value to key and translates product. Removed wildcard import

Commit:
BroadleafCommerce/QA#4771
- refactoring

Commit:
BroadleafCommerce/QA#4771
- refactoring

oleksiimiroshnyk reviewed (changes requested):
there is a discussion going on in helpscout if this is reasonable or not to do this change.
So let;'s wait and don't merge till then
